### PR TITLE
Release 3.2.0 (prod)

### PR DESCRIPTION
**Pull request by cthtuf**

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ meta:
 
 # Release notes
 
+## Chainstack 3.2
+
+*August 23, 2022*
+
+### What’s new
+
+* **Protocols**. [Gnosis](/operations/gnosis/networks) support.
```
Comment (cthtuf) [inline docs/release-notes.md:17]: Something should be added here

**Review by akegaviar — CHANGES_REQUESTED** [review]
Please include https://github.com/chainstack/docs/pull/493

**Comment by cthtuf** [comment]
> Please include #493

fixed

**Review by akegaviar — APPROVED** [review]

**Review by PekopT — APPROVED** [review]
LGTM

**Pull request merged**